repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/proxy/PassThruTest.groovy

```diff
@@ -244,12 +238,11 @@ class PassThruTest extends ReposeValveTest {
         'Cache-Control'       | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
         'Content-Encoding'    | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
         'Content-Language'    | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
-        'Content-Length'      | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
         'Content-Location'    | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
         'Content-MD5'         | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
         'Content-Range'       | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
         'Content-Type'        | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
-        'Date'                | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc'
+//        'Date'                | 'something/1.0; q=0.5, another/0.8 ;q=0.8=0.3 ; a=b=c ; abc' //This one does go through, but a second date is applied
```

Comment:
It looks like Jetty adds the second `Date` header. Jetty can be configured to add that header or not to add it, but I'm not sure if there is a way to tell Jetty to add the header only if it does not already exist.